librz/bin/format/dex/dex.c

```diff
@@ -34,6 +34,133 @@ void rz_bin_dex_free(RzBinDexObj *dex) {
 	free(dex);
 }
 
+static bool dex_new_buf_aux(RzBinDexObj *bin, struct dex_header_t *dexhdr) {
```

Comment:
This sequence could be compressed in a macro

Reply:
@wargio Is refactoring the `dex` plugin, so this code is temporary.